src/Razor/test/Microsoft.VisualStudio.LanguageServer.ContainedLanguage.Test.Common/TestVirtualDocument.cs

```diff
@@ -1,12 +1,10 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT license. See License.txt in the project root for license information.
 
-#nullable disable
-
-using System;
 using Microsoft.VisualStudio.Text;
+using System;
 
-namespace Microsoft.VisualStudio.LanguageServer.ContainedLanguage
+namespace Microsoft.VisualStudio.LanguageServer.ContainedLanguage.Test.Common
```

Comment:
Moved this test class to a different namespace/project where it can be more widely used